devito/data/data.py

```diff
@@ -26,6 +27,8 @@ class Data(np.ndarray):
         If the i-th entry is True, then the i-th array dimension uses modulo indexing.
     allocator : MemoryAllocator, optional
         Used to allocate memory. Defaults to ``ALLOC_FLAT``.
+    distributor : Distributor, optional
+        The distributor attached to the grid on which the Data object sits.
```

Comment:
shouldn't talk about "grid" here . There's no concept of grid in ``data``

Reply:
Updated.